### PR TITLE
Update Set-PSReadlineOption.md

### PR DESCRIPTION
**Pull request by doctordns**
changed fencing to PowerShell, and fixed a typo.

Version(s) of document impacted
------------------------------
- [x ] Impacts 6.next document
- [ ] Impacts 6 document
- [ ] Impacts 5.1 document
- [ ] Impacts 5.0 document
- [ ] Impacts 4.0 document
- [ ] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [ ] The documented feature was introduced in version (list version here) of PowerShell
- [ ] This issue only shows up in version (list version(s) here) of the document
- [ ] This PR partially fixes the issue, and issue #<insert here> tracks the remaining work

### CODE REVIEW

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[d187563](https://github.com/doctordns/PowerShell-Docs/pull/3607/commits/d187563de1eb0876e99a7eefd2f6a851ffc6c4b0)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 8 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[d187563](https://github.com/doctordns/PowerShell-Docs/pull/3607/commits/d187563de1eb0876e99a7eefd2f6a851ffc6c4b0)_: 

:clock10: **Incremental building**: average incremental building time is _12 min(s) 52 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[d187563](https://github.com/doctordns/PowerShell-Docs/pull/3607/commits/d187563de1eb0876e99a7eefd2f6a851ffc6c4b0)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/6/PSReadLine/Set-PSReadlineOption.md](https://github.com/doctordns/PowerShell-Docs/blob/patch-1/reference/6/PSReadLine/Set-PSReadlineOption.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/psreadline/set-psreadlineoption?view=powershell-6&branch=pr-en-us-3607) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C25%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201901251657310061-3607%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=zTh9OK6ry4W1KHhw%2BHV2kCE%2FQXMUB%2FmFQYqgxCovmtY%3D&st=2019-01-25T17%3A08%3A08Z&se=2019-02-25T17%3A13%3A08Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/6/PSReadLine/Set-PSReadlineOption.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Customizes the behavior of command line editing in PSReadLine.
 
 ### OptionsSet
 
-```
+```powershell
```
Comment (sdwheeler) [inline reference/6/PSReadLine/Set-PSReadlineOption.md]: This is not PowerShell code it is a syntax block. We don't use the "powershell" tag on the code fence for syntax blocks. See https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/contributing/5-FORMATTING-CODE.md#formatting-illustrative-examples

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[3dc1976](https://github.com/doctordns/PowerShell-Docs/pull/3607/commits/3dc1976287a70dcb33d719237e61920fea359a5b)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 10 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[3dc1976](https://github.com/doctordns/PowerShell-Docs/pull/3607/commits/3dc1976287a70dcb33d719237e61920fea359a5b)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/6/PSReadLine/Set-PSReadlineOption.md](https://github.com/doctordns/PowerShell-Docs/blob/patch-1/reference/6/PSReadLine/Set-PSReadlineOption.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/psreadline/set-psreadlineoption?view=powershell-6&branch=pr-en-us-3607) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C25%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201901252256116591-3607%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=Cme6Cb%2BFPra43yGgEvzL%2BJVK6u1gX923eZuMOV1DlaY%3D&st=2019-01-25T23%3A04%3A19Z&se=2019-02-25T23%3A09%3A19Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[455168f](https://github.com/doctordns/PowerShell-Docs/pull/3607/commits/455168f3c7b76c108ad7af545263dc50257fcebb)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 9 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[455168f](https://github.com/doctordns/PowerShell-Docs/pull/3607/commits/455168f3c7b76c108ad7af545263dc50257fcebb)_: 

:clock10: **Incremental building**: average incremental building time is _12 min(s) 59 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[455168f](https://github.com/doctordns/PowerShell-Docs/pull/3607/commits/455168f3c7b76c108ad7af545263dc50257fcebb)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/6/PSReadLine/Set-PSReadlineOption.md](https://github.com/doctordns/PowerShell-Docs/blob/patch-1/reference/6/PSReadLine/Set-PSReadlineOption.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/psreadline/set-psreadlineoption?view=powershell-6&branch=pr-en-us-3607) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C26%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201901252350375578-3607%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=EkKsgMR5G95%2Fi7dQCJ3aGT3szGwwMp3JDblD76eVR4A%3D&st=2019-01-25T23%3A59%3A42Z&se=2019-02-26T00%3A04%3A42Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[f160d8f](https://github.com/doctordns/PowerShell-Docs/pull/3607/commits/f160d8f9762eed6b4bc14e138bdb2344cab5826a)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 8 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[f160d8f](https://github.com/doctordns/PowerShell-Docs/pull/3607/commits/f160d8f9762eed6b4bc14e138bdb2344cab5826a)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 8 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[f160d8f](https://github.com/doctordns/PowerShell-Docs/pull/3607/commits/f160d8f9762eed6b4bc14e138bdb2344cab5826a)_: 

:clock10: **Incremental building**: average incremental building time is _12 min(s) 52 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[845cbe3](https://github.com/doctordns/PowerShell-Docs/pull/3607/commits/845cbe37b021c7b3e016c6e423de4ea0c524e6fc)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 8 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[845cbe3](https://github.com/doctordns/PowerShell-Docs/pull/3607/commits/845cbe37b021c7b3e016c6e423de4ea0c524e6fc)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/6/PSReadLine/Set-PSReadlineOption.md](https://github.com/doctordns/PowerShell-Docs/blob/patch-1/reference/6/PSReadLine/Set-PSReadlineOption.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/psreadline/set-psreadlineoption?view=powershell-6&branch=pr-en-us-3607) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C28%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201901281618369316-3607%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=sBMdX0BK8iOBNGv8e2XXMai9Ww1n3dDR5%2BF1WhZcCYI%3D&st=2019-01-28T16%3A36%3A36Z&se=2019-02-28T16%3A41%3A36Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[8eb929d](https://github.com/doctordns/PowerShell-Docs/pull/3607/commits/8eb929da58474c4f87d1d3b0f05d0285c3d9b63f)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 9 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[8eb929d](https://github.com/doctordns/PowerShell-Docs/pull/3607/commits/8eb929da58474c4f87d1d3b0f05d0285c3d9b63f)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 9 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[8eb929d](https://github.com/doctordns/PowerShell-Docs/pull/3607/commits/8eb929da58474c4f87d1d3b0f05d0285c3d9b63f)_: 

:clock10: **Incremental building**: average incremental building time is _12 min(s) 53 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[8eb929d](https://github.com/doctordns/PowerShell-Docs/pull/3607/commits/8eb929da58474c4f87d1d3b0f05d0285c3d9b63f)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/6/PSReadLine/Set-PSReadlineOption.md](https://github.com/doctordns/PowerShell-Docs/blob/patch-1/reference/6/PSReadLine/Set-PSReadlineOption.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/psreadline/set-psreadlineoption?view=powershell-6&branch=pr-en-us-3607) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C28%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201901281651589141-3607%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=pITPqAhIaSgakLznGjfc7%2Btunw%2FWaAfWPohK3q%2BGQSM%3D&st=2019-01-28T17%3A01%3A15Z&se=2019-02-28T17%3A06%3A15Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**